KubeArmor/BPF/tests/Makefile

```
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 Authors of KubeArmor
+
+
+include ../Makefile.vars
+
+LIBBPF = $(CURDIR)/../libbpf
+
+Cfiles =  $(wildcard checks/*.c)
+files = $(Cfiles:.c=)
+
+.PHONY: all
+all: compile-all check
+
+.PHONY: compile-all
+compile-all: checks/*.c
+	@for i in $(files) ; do \
+	echo "Compiling eBPF bytecode: $(GREEN)$$i.c$(NC) ..." && \
+	$(CL) $(KF) -DMONITOR_HOST_AND_CONTAINER -c $$i.c -o -| llc -march=bpf -mcpu=probe -filetype=obj -o $$i.bpf.o;\
```

```suggestion
	$(CL) $(KF) -DMONITOR_HOST_AND_CONTAINER -DSECURE_PATH_UNLINK -c $$i.c -o -| llc -march=bpf -mcpu=probe -filetype=obj -o $$i.bpf.o;\
```

We add the flag to compilation if supported else not.